app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,40 @@
+// Place all the styles related to the Posts controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.topnav {
+  background-color: #ff6600;
+  overflow: hidden;
+  font-size: 17px;
```

Comment:
we alphabetize the CSS properties so that you avoid duplication. not sure if all editors do this, but in atom, you can select them and hit F5 (or function + F5). so this would be rewritten as:

```css
  background-color: #ff6600;
  font-size: 17px;
  overflow: hidden;
```